js_src/pid_display.js

```diff
@@ -156,13 +165,17 @@ app.MessageStructure.prototype.decorateInternal = function(element) {
 
 /**
  * Check if we can decorate an element.
- * @param {Element} element the dom element to check.
+ * @param {Element} element The DOM element to check.
+ * @return {!boolean} True if we can decorate the element.
  */
 app.MessageStructure.prototype.canDecorate = function(element) {
   return element.tagName == 'DIV';
 };
 
 
+/**
+ * @param {!Array.<app.MessageField>} fields
```

Comment:
Can we be consistent please, either Array.< or Array< throughout.

Reply:
Hmm, perhaps just my suggestion in a comment here :flushed: : https://github.com/OpenLightingProject/rdm-app/pull/128#discussion_r172020770

Reply:
Done